pandas-stubs/io/orc.pyi

```diff
@@ -1,7 +1,26 @@
-from pandas.core.frame import DataFrame
+from typing import (
+    Any,
+    Literal,
+)
 
-from pandas._typing import FilePathOrBuffer
+from pandas import DataFrame
+
+from pandas._typing import (
+    FilePath,
+    ReadBuffer,
+    WriteBuffer,
+)
 
 def read_orc(
-    path: FilePathOrBuffer, columns: list[str] | None = ..., **kwargs
+    path: FilePath | ReadBuffer[bytes],
+    columns: list[str] | None = ...,
+    **kwargs: Any,
 ) -> DataFrame: ...
+def to_orc(
```

Comment:
`pandas.io.to_orc()` is not documented, but v1.5 will have `DataFrame.to_orc()`, so can you put the stub there?